client/jetpack-connect/remote-credentials.js

```diff
@@ -5,7 +5,7 @@
 import classnames from 'classnames';
 import React, { Component, Fragment } from 'react';
 import config from 'config';
-import Gridicon from 'gridicons';
+import Gridicon from 'components/gridicon';
```

Comment:
I think this is supposed to move to `External dependencies` now

Reply:
Good call! Thanks @p-jackson, I've just updated it.